backend/services/deployments/api/http/routing.go

```diff
@@ -113,34 +116,37 @@ func NewRouter(
 	ds store.DataStore,
 	cfg *Config,
 ) http.Handler {
+	if cfg == nil {
+		cfg = NewConfig()
+	}
 	router := routing.NewMinimalGinRouter()
 	// Create and configure API handlers
 	//
 	// Encode base64 secret in either std or URL encoding ignoring padding.
 	deploymentsHandlers := NewDeploymentsApiHandlers(
 		ds, new(view.RESTView), app, cfg,
 	)
+	router.Use(requestid.Middleware())
+	accesslogDefault := accesslog.Middleware()
+	accesslogSupressed := accesslog.AccessLogger{
+		DisableLog: func(c *gin.Context) bool {
+			if c.Writer.Status()/100 == 2 {
```

Comment:
```suggestion
			if c.Writer.Status()-200 < 100 {
```
(or something like that I would suggest)
you want explicitly 200-299 range? or 1xx fits return true as well?
division always costs more than multiplication and both more than subtraction.

Reply:
Right! Indeed a 100 status code would never reach the access logs either! Let's drop even the subtraction and do `status < 300`.